frontend/components/Player/Player.jsx

```diff
@@ -0,0 +1,13 @@
+const Player = ({ player }) => {
+    return (
+        <div>
+            <h1>Members</h1>
+            {player.map((member, idx) => {
+                let email = member.email;
```

Comment:
*nbitpick. If you're not changing the variable name, you should use const. 'let' implies that the variable will be changing.